components/learn/boxes/Boxes.js

```diff
@@ -65,15 +65,15 @@ export default function Boxes(props) {
               <Col xs={12} md={4} lg={4} className={styles.boxCol}>
                 <div className={styles.cardWrapper}>
                   <div>
-                    <h3>Integrations</h3>
+                    <h3>Integration</h3>
                     <div className={styles.cardDescription}>
                       <div className={styles.content}>
                         <p className={styles.title}>
                           <a href={`${prefix}/learn2/#`} className={styles.titleLink}>
                             Integration examples
```

Comment:
This should be `Pre-built integrations` and this should be second title.

Reply:
Updated.